bb.edn

```diff
@@ -0,0 +1,62 @@
+{:min-bb-version "0.8.0"
+ :pods {;since babashka 0.8.0, can only be declared in bb.edn
+        org.babashka/tools-deps-native {:version "0.0.5"}   ;; for babashka build
+        clj-kondo/clj-kondo            {:version "2022.10.05"}}
+ :deps  {datahike/bb {:local/root "bb"}}
+ :tasks {; tools
+
+         fmt {:doc "Format source files"
+              :task tasks.format/fix}
+
+         docs {:doc "Get a preview on the clojure docs"
```

Comment:
The task only throws some server errors now, but that might be something temporary

Reply:
right, still I don't know why they are failing, because sometimes it works fine. seems a bit random. the old script wasn't fully functional as well. so just drop it I'd say.